litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -727,3 +727,18 @@ spec:
       targetPort: 27017
   selector:
     component: database
+---
```

Comment:
Do we need above service then?

Reply:
yes, headless service is not accessible outside. It's different than the normal service